service/controller/cluster.go

```diff
@@ -1,17 +1,38 @@
 package controller
```

Comment:
I moved the resource set initialization into this file from `service/controller/cluster_resource_set.go` since it's related logic and to align with AWS. Same for the other two controllers.